From 55e9efcac8a7b6c04a255e9d6de61d6a5b7ed21e Mon Sep 17 00:00:00 2001
From: Jan Nieuwenhuizen <janneke@gnu.org>
Date: Fri, 3 Dec 2010 22:37:19 +0100
Subject: [PATCH] midi2ly: bump version in output to 2.7.38 (oldest supported).

This makes midi2ly output actually processable without problems by
the latest lilypond.
---
 scripts/midi2ly.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/midi2ly.py b/scripts/midi2ly.py
index e0a00ae..eb15312 100644
--- a/scripts/midi2ly.py
+++ b/scripts/midi2ly.py
@@ -826,7 +826,7 @@ def convert_midi (in_file, out_file):
 
     
     s = ''
-    s = tag + '\n\\version "2.7.18"\n\n'
+    s = tag + '\n\\version "2.7.38"\n\n'
     for i in range (len (tracks)):
         s = s + dump_track (tracks[i], i)
 
-- 
1.7.1

